src/client/components/features/Accompagnement/Rechercher/Résultat/RésultatRechercherAccompagnement.tsx

```diff
@@ -80,25 +81,29 @@ export function RésultatRechercherAccompagnement({ établissement }: RésultatR
 		</>;
 	}
 
+	function getAdressFormated(adresse: EtablissementAccompagnement.Adresse) {
```

Comment:
```suggestion
	function getFormattedAddress(adresse: EtablissementAccompagnement.Adresse) {
```
ou 
```suggestion
	function getAdresseFormatted(adresse: EtablissementAccompagnement.Adresse) {
```
voire même
```suggestion
	function formatAdresse(adresse: EtablissementAccompagnement.Adresse) {
```
vu que c'est pas un getter mais simplement une fonction qui formatte ce que tu lui donne en paramètre
**Voire, vu que c'est de la logique d'affichage**
```suggestion
	function Address(adresse: EtablissementAccompagnement.Adresse) {
```
(On le voit pas, là comme ça, mais c'est supposé être un composant React)

Reply:
Et si tu le passe en composant, ça veut dure que tu peux inliner le `span` et la classe direct dedans